atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/persistentlock/LockStoreImpl.java

```diff
@@ -102,16 +102,16 @@ public static LockStore create(KeyValueService kvs) {
     }
 
     public static LockStore create(KeyValueService kvs, boolean initializeAsync) {
-        LockStoreImpl lockStore = createImplForTest(kvs, initializeAsync);
+        LockStoreImpl lockStore = createImpl(kvs, initializeAsync);
         return lockStore.wrapper.isInitialized() ? lockStore : lockStore.wrapper;
     }
 
     @VisibleForTesting
     static LockStoreImpl createImplForTest(KeyValueService kvs) {
-        return createImplForTest(kvs, AtlasDbConstants.DEFAULT_INITIALIZE_ASYNC);
```

Comment:
This one should be for test! That's how I imagine the renaming happened in the first place 🥇